result/array-bracket-spacing.js

```diff
@@ -0,0 +1,118 @@
+/*eslint-env es6*/
+
+var arr = [ 'foo', 'bar' ];
```

Comment:
__[development][eslint]__
* Error - 'arr' is defined but never used (no-unused-vars)